docx/header.py

```diff
@@ -8,12 +8,11 @@
     absolute_import, division, print_function, unicode_literals
```

Comment:
This is great @eupharis :)

I'll have a look this evening and move this along :)